src/common/paddle/recruiter/processing.ts

```diff
@@ -141,6 +141,8 @@ export const createOpportunitySubscription = async ({
         flags: updateFlagsStatement<OpportunityJob>({
           batchSize: priceCustomData.batch_size,
           plan: price.id,
+          reminders: priceCustomData.reminders,
```

Comment:
This is stored on database level? (`priceCustomData` I mean)

Reply:
No it comes from paddle custom data objects.